packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -554,12 +562,13 @@ export abstract class BaseSegment implements ISegment {
 		return false;
 	}
 
-	protected addSerializedProps(jseg: IJSONSegment) {
+	protected addSerializedProps(jseg: IJSONSegment): void {
 		if (this.properties) {
 			jseg.props = { ...this.properties };
 		}
 	}
 
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Let's document why we can't convert this yet and/or leave a TODO to come back later and clean this up.